tests/tasks/cloud/test_flow_run.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from unittest.mock import MagicMock
+
+from prefect.tasks.cloud.flow_run import FlowRunTask
+
+
+@pytest.fixture()
+def client(monkeypatch):
+    cloud_client = MagicMock(
+        graphql=MagicMock(
+            return_value=MagicMock(
+                data=MagicMock(flow=[MagicMock(id="abc123"), MagicMock(id="def456")])
+            )
+        ),
+        create_flow_run=MagicMock(return_value="xyz890"),
+    )
+    monkeypatch.setattr(
+        "prefect.tasks.cloud.flow_run.Client", MagicMock(return_value=cloud_client),
+    )
+    yield cloud_client
+
+
+class TestFlowRunTask:
+    def test_initialization(self):
```

Comment:
Could you throw a few task-generic kwargs into this test? e.g., "name", "tags" -- this will ensure that `super().__init__()` was called correctly